sed/calibrator/energy.py

```diff
@@ -877,6 +881,20 @@ def append_energy_axis(
         else:
             raise NotImplementedError
 
+        # apply bias offset
+        scale_sign: Literal[-1, 1] = -1 if calibration["energy_scale"] == "binding" else 1
+        if bias_voltage is not None:
+            df[energy_column] = df[energy_column] + scale_sign * bias_voltage
+        elif self._config["dataframe"]["bias_column"] in df.columns:
+            df = dfops.offset_by_other_columns(
+                df=df,
+                target_column=energy_column,
+                offset_columns=self._config["dataframe"]["bias_column"],
+                weights=scale_sign,
+            )
+        else:
+            print("Sample bias data not found or provided. Calibrated energy might be incorrect.")
```

Comment:
Perhaps better to throw an error as this is not intended behavior?

Reply:
As mentioned below, I don't think this is a critical error. It still provides a valid calibration, just with an energy offset. Otherwise, no calibration is possible if you don't know the bias.